src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -793,15 +830,35 @@ private MeasureInfo CreateGenericMeasureInfo(Statistics stat, bool shouldUseGene
         {
             double? sum = null;
             double? average = null;
+            double? stdDeviation = null;
             object max = null;
             object min = null;
 
             if (!_nonNumericError)
             {
                 if (_measureSum)
                     sum = stat.sum;
+
                 if (_measureAverage && stat.count > 0)
                     average = stat.sum / stat.count;
+
+                if(_measureStdDeviation && !_measureAverage) {
+                    var message = "StdDeviation was requested and requires the average to be calculated, please add the -Average switch";
```

Comment:
messages should be in the corresponding .resx file.  See line 537 as an example.